dbs/settings.js

```diff
@@ -100,5 +101,12 @@ migrations = [
       INSERT INTO settings (key, value) VALUES ('start_page_background_image', '');
       PRAGMA user_version = 2
     `, cb)
+  },
+  // version 3
+  function (cb) {
+    db.exec(`
+      INSERT INTO settings (key, value) VALUES ('auto_redirect_to_dat', 1);
+      PRAGMA user_version = 3
+    `, cb)
```

Comment:
We actually don't need to do this version 3 because of the `defaultSettings`, so all we need to do is have the addition at line 27 and this bit can be removed